template-literals/01-intro.mdx

```diff
@@ -62,4 +62,5 @@ import nightOwl from "prism-react-renderer/themes/nightOwl"
   ]}
 />
 ---
-TODO: Exercise / Challenge Time
\ No newline at end of file
+TODO: Exercise / Challenge Time
+NOTES: Why is it useful? --> Early mention of JSX?
```

Comment:
Do you think it would be more useful to not have an exercise here yet - but once we get into functions / destructuring arguments / etc - a could time to pull on information from here?

as a 'standalone let's play with string literals' does feel a little too basic.

Reply:
I think that works. This is probably fine as an example.